ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge3A.cfg

```diff
@@ -299,18 +299,6 @@ ip extcommunity-list ECL-EVPN-SOO permit soo 192.168.42.6:404
 ip prefix-list PL-LOOPBACKS-EVPN-OVERLAY
    seq 10 permit 192.168.42.0/24 eq 32
 !
-route-map RM-BGP-UNDERLAY-PEERS-IN permit 40
```

Comment:
curious what happens to WAN-PATH / PL-WAN-HA-PEER-PREFIXES when they are generated in this route-map and we delete the peer-group.
I think the code currently does not cater for that?